### PR TITLE
DEV-686 Table component redesign with light/dark mode

### PR DESCRIPTION
**Pull request by willmccormick24**
I added the ability to sort the data by headers alphanumerically. Provided simple mock data to visualize table.

Light Mode:
Screenshot 2023-09-19 at 12 13 00 PM

Dark Mode:
Screenshot 2023-09-19 at 12 12 09 PM

### CODE REVIEW

File: src/components/common/table.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+interface TableProps {
+	headers: string[];
+	data: any[];
+}
+
+const Table: React.FC<TableProps> = ({ headers, data }) => {
```
Comment (Ericky14) [inline src/components/common/table.tsx:9]: Please don't make your own table, table components are complicated and we already have a table component

**Comment by willmccormick24** [comment]
The new table also reskins the Assets Page, which is the only implementation of a table after the reskin.

Light Mode:
Screenshot 2023-09-28 at 5 47 59 PM

Dark Mode:
Screenshot 2023-09-28 at 5 47 33 PM

File: src/colors.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//Colors for Redesign
+
+const colors = {
```
Comment (nicolam1) [inline src/colors.tsx:3]: I keep seeing this file in the PRs. I think you are pointing to the wrong branch maybe

**Review by nicolam1 — CHANGES_REQUESTED** [review]
Fix commit or PR

File: src/colors.tsx (hunk 2 above)
Comment (willmccormick24) [reply src/colors.tsx:3]: @nicolam1 that's the new color scheme for the redesign I made initially with the new colors from FIgma. It's referenced in the `src/theme.tsx` file just to make the code a little more readable. I can get rid of it and just put the colors directly in the theme file if you want? The `colors.tsx` file is already in master though, this branch is able to merge with master.

Comment (Ericky14) [reply src/colors.tsx:3]: @willmccormick24 If this was in master then we wouldn't see this change in the DIFF, maybe rebase the branch on master and push again

File: src/pages/stake/table.tsx [hunk 3]
```diff
@@ -297,11 +297,13 @@ const Table = observer<TableProps>(({ active, className, tableRowClassName }) =>
 	const noDataFound = <div className="no_data_table"> No data found </div>;
 	const loader = <CircularProgress />;
 
+	const data = [['Test'], ['Test'], ['Test']];
```
Comment (Ericky14) [inline src/pages/stake/table.tsx:300]: Remove?

**Review by Ericky14 — CHANGES_REQUESTED** [review]
You messed up styles in other places in the app

Screenshot 2023-10-02 at 11 22 48 AM
Screenshot 2023-10-02 at 11 23 56 AM

**Comment by willmccormick24** [comment]
In the most recent commit, I addressed the comments and I fixed the button issues in DEV-672 pages redesign then made this branch point to that, so should be good to go 👍

**Pull request closed without merging**